README.md

```diff
@@ -1 +1,50 @@
-# Bubble-Sort
\ No newline at end of file
+# Bubble Sort
+
+This is a collaborative project about building a bubble sort algorithm.
+
+![Bubble Sort Animation](Bubble-sort-example.gif)
+
+The purpose of the project is to implement Bubble sort algorithm with Ruby using yield and passing a block to a method. 
+
+## Development
+
+* Run script
+```
+ruby bubble_sort.rb
+```
```

Comment:
[Optional]
Replace line 12 with the following line. It will give you a fancy look.
```
```ruby
```
```